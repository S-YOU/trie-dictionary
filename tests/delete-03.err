nodes: 2
nodes: 3
nodes: 4
nodes: 5
nodes: 7
nodes: 9
nodes: 10
nodes: 11
nodes: 13
nodes: 15
nodes: 16
nodes: 18
nodes: 20
nodes: 21
nodes: 22
nodes: 23
nodes: 25
nodes: 26
nodes: 27
nodes: 29
nodes: 31
nodes: 32
nodes: 33
nodes: 34
nodes: 36
nodes: 38
nodes: 39
nodes: 41
nodes: 42
nodes: 44
nodes: 46
nodes: 48
nodes: 49
nodes: 50
nodes: 52
nodes: 54
nodes: 56
nodes: 57
nodes: 58
nodes: 59
nodes: 60
nodes: 61
nodes: 63
nodes: 64
nodes: 66
nodes: 67
nodes: 69
nodes: 71
nodes: 72
nodes: 74
nodes: 76
nodes: 77
nodes: 78
nodes: 80
nodes: 82
nodes: 83
nodes: 84
nodes: 86
nodes: 87
nodes: 89
nodes: 90
nodes: 92
nodes: 94
nodes: 95
nodes: 97
nodes: 99
nodes: 101
nodes: 103
nodes: 105
nodes: 107
nodes: 108
nodes: 110
nodes: 111
nodes: 113
nodes: 114
nodes: 116
nodes: 117
nodes: 118
nodes: 120
nodes: 121
nodes: 123
nodes: 125
nodes: 126
nodes: 128
nodes: 126
nodes: 127
nodes: 128
nodes: 129
nodes: 130
nodes: 131
nodes: 133
nodes: 134
nodes: 135
nodes: 137
nodes: 139
nodes: 140
nodes: 142
nodes: 144
nodes: 145
nodes: 146
nodes: 148
nodes: 149
nodes: 151
nodes: 152
nodes: 153
nodes: 154
nodes: 156
nodes: 157
nodes: 158
nodes: 159
nodes: 161
nodes: 163
nodes: 165
nodes: 167
nodes: 168
nodes: 169
nodes: 171
nodes: 173
nodes: 175
nodes: 176
nodes: 178
nodes: 179
nodes: 180
nodes: 182
nodes: 184
nodes: 186
nodes: 185
nodes: 186
nodes: 185
nodes: 187
nodes: 188
nodes: 189
nodes: 191
nodes: 193
nodes: 194
nodes: 195
nodes: 196
nodes: 197
nodes: 196
nodes: 197
nodes: 198
nodes: 200
nodes: 201
nodes: 202
nodes: 203
nodes: 204
nodes: 205
nodes: 207
nodes: 208
nodes: 206
nodes: 204
nodes: 203
nodes: 201
nodes: 203
nodes: 205
nodes: 207
nodes: 208
nodes: 209
nodes: 211
nodes: 212
nodes: 213
nodes: 214
nodes: 215
nodes: 216
nodes: 217
nodes: 218
nodes: 220
nodes: 221
nodes: 222
nodes: 224
nodes: 226
nodes: 227
nodes: 229
nodes: 231
nodes: 232
nodes: 234
nodes: 236
nodes: 238
nodes: 239
nodes: 241
nodes: 243
nodes: 245
nodes: 247
nodes: 248
nodes: 249
nodes: 251
nodes: 253
nodes: 254
nodes: 256
nodes: 257
nodes: 258
nodes: 259
nodes: 261
nodes: 262
nodes: 263
nodes: 264
nodes: 265
nodes: 267
nodes: 269
nodes: 270
nodes: 268
nodes: 269
nodes: 271
nodes: 272
nodes: 273
nodes: 275
nodes: 276
nodes: 277
nodes: 279
nodes: 281
nodes: 282
nodes: 284
nodes: 285
nodes: 287
nodes: 288
nodes: 290
nodes: 292
nodes: 294
nodes: 296
nodes: 298
nodes: 300
nodes: 301
nodes: 302
nodes: 304
nodes: 305
nodes: 307
nodes: 308
nodes: 309
nodes: 310
nodes: 311
nodes: 312
nodes: 313
nodes: 315
nodes: 317
nodes: 319
nodes: 320
nodes: 322
nodes: 324
nodes: 325
nodes: 327
nodes: 329
nodes: 330
nodes: 332
nodes: 333
nodes: 335
nodes: 337
nodes: 338
nodes: 340
nodes: 341
nodes: 343
nodes: 344
nodes: 345
nodes: 346
nodes: 347
nodes: 349
nodes: 348
nodes: 349
nodes: 349
nodes: 350
nodes: 351
nodes: 353
nodes: 354
nodes: 355
nodes: 357
nodes: 359
nodes: 360
nodes: 361
nodes: 363
nodes: 364
nodes: 365
nodes: 366
nodes: 368
nodes: 369
nodes: 370
nodes: 372
nodes: 374
nodes: 375
nodes: 373
nodes: 374
nodes: 376
nodes: 378
nodes: 380
nodes: 381
nodes: 382
nodes: 383
nodes: 384
nodes: 386
nodes: 388
nodes: 389
nodes: 391
nodes: 393
nodes: 394
nodes: 395
nodes: 396
nodes: 397
nodes: 398
nodes: 399
nodes: 401
nodes: 403
nodes: 404
nodes: 406
nodes: 407
nodes: 408
nodes: 410
nodes: 411
nodes: 413
nodes: 415
nodes: 417
nodes: 419
nodes: 421
nodes: 423
nodes: 424
nodes: 425
nodes: 426
nodes: 427
nodes: 428
nodes: 430
nodes: 432
nodes: 433
nodes: 432
nodes: 433
nodes: 431
nodes: 430
nodes: 429
nodes: 428
nodes: 429
nodes: 430
nodes: 431
nodes: 432
nodes: 433
nodes: 435
nodes: 436
nodes: 437
nodes: 439
nodes: 441
nodes: 443
nodes: 444
nodes: 446
nodes: 448
nodes: 449
nodes: 451
nodes: 453
nodes: 454
nodes: 456
nodes: 457
nodes: 459
nodes: 460
nodes: 462
nodes: 464
nodes: 465
nodes: 466
nodes: 467
nodes: 469
nodes: 471
nodes: 473
nodes: 475
nodes: 476
nodes: 477
nodes: 478
nodes: 480
nodes: 481
nodes: 482
nodes: 483
nodes: 485
nodes: 487
nodes: 488
nodes: 490
nodes: 491
nodes: 493
nodes: 494
nodes: 495
nodes: 496
nodes: 498
nodes: 499
nodes: 501
nodes: 503
nodes: 504
nodes: 505
nodes: 506
nodes: 508
nodes: 509
nodes: 511
nodes: 509
nodes: 511
nodes: 512
nodes: 513
nodes: 515
nodes: 516
nodes: 518
nodes: 520
nodes: 521
nodes: 522
nodes: 521
nodes: 519
nodes: 517
nodes: 515
nodes: 517
nodes: 516
nodes: 514
nodes: 513
nodes: 512
nodes: 511
nodes: 509
nodes: 510
nodes: 511
nodes: 513
nodes: 514
nodes: 515
nodes: 516
nodes: 518
nodes: 519
nodes: 520
nodes: 521
nodes: 522
nodes: 524
nodes: 525
nodes: 526
nodes: 527
nodes: 528
nodes: 529
nodes: 530
nodes: 531
nodes: 533
nodes: 534
nodes: 536
nodes: 538
nodes: 540
nodes: 542
nodes: 543
nodes: 545
nodes: 547
nodes: 549
nodes: 550
nodes: 551
nodes: 552
nodes: 553
nodes: 554
nodes: 556
nodes: 558
nodes: 560
nodes: 562
nodes: 564
nodes: 565
nodes: 566
nodes: 567
nodes: 569
nodes: 571
nodes: 573
nodes: 575
nodes: 576
nodes: 578
nodes: 579
nodes: 580
nodes: 582
nodes: 583
nodes: 585
nodes: 587
nodes: 589
nodes: 590
nodes: 591
nodes: 593
nodes: 595
nodes: 596
nodes: 597
nodes: 598
nodes: 600
nodes: 602
nodes: 604
nodes: 605
nodes: 607
nodes: 608
nodes: 609
nodes: 610
nodes: 612
nodes: 614
nodes: 615
nodes: 616
nodes: 617
nodes: 618
nodes: 620
nodes: 621
nodes: 622
nodes: 624
nodes: 626
nodes: 627
nodes: 629
nodes: 630
nodes: 632
nodes: 633
nodes: 635
nodes: 636
nodes: 638
nodes: 639
nodes: 641
nodes: 643
nodes: 644
nodes: 646
nodes: 647
nodes: 648
nodes: 650
nodes: 652
nodes: 653
nodes: 655
nodes: 656
nodes: 657
nodes: 659
nodes: 660
nodes: 661
nodes: 663
nodes: 664
nodes: 666
nodes: 667
nodes: 669
nodes: 670
nodes: 671
nodes: 673
nodes: 675
nodes: 677
nodes: 678
nodes: 679
nodes: 681
nodes: 682
nodes: 683
nodes: 685
nodes: 686
nodes: 688
nodes: 689
nodes: 690
nodes: 691
nodes: 693
nodes: 695
nodes: 696
nodes: 698
nodes: 699
nodes: 700
nodes: 702
nodes: 704
nodes: 706
nodes: 705
nodes: 703
nodes: 702
nodes: 701
nodes: 699
nodes: 698
nodes: 696
nodes: 695
nodes: 694
nodes: 693
nodes: 692
nodes: 691
nodes: 689
nodes: 687
nodes: 686
nodes: 685
nodes: 684
nodes: 683
nodes: 685
nodes: 687
nodes: 689
nodes: 690
nodes: 691
nodes: 693
nodes: 695
nodes: 697
nodes: 698
nodes: 699
nodes: 701
nodes: 702
nodes: 704
nodes: 705
nodes: 707
nodes: 708
nodes: 709
nodes: 710
nodes: 712
nodes: 713
nodes: 714
nodes: 716
nodes: 717
nodes: 718
nodes: 719
nodes: 720
nodes: 721
nodes: 723
nodes: 725
nodes: 727
nodes: 728
nodes: 729
nodes: 730
nodes: 731
nodes: 733
nodes: 735
nodes: 737
nodes: 738
nodes: 740
nodes: 742
nodes: 743
nodes: 744
nodes: 745
nodes: 746
nodes: 747
nodes: 749
nodes: 750
nodes: 752
nodes: 753
nodes: 755
nodes: 756
nodes: 757
nodes: 759
nodes: 760
nodes: 761
nodes: 762
nodes: 764
nodes: 766
nodes: 767
nodes: 768
nodes: 770
nodes: 772
nodes: 774
nodes: 775
nodes: 776
nodes: 777
nodes: 778
nodes: 780
nodes: 782
nodes: 784
nodes: 785
nodes: 787
nodes: 788
nodes: 789
nodes: 791
nodes: 793
nodes: 794
nodes: 796
nodes: 797
nodes: 798
nodes: 800
nodes: 802
nodes: 804
nodes: 806
nodes: 807
nodes: 809
nodes: 810
nodes: 811
nodes: 813
nodes: 815
nodes: 817
nodes: 818
nodes: 819
nodes: 820
nodes: 822
nodes: 823
nodes: 824
nodes: 825
nodes: 827
nodes: 829
nodes: 830
nodes: 831
nodes: 832
nodes: 834
nodes: 835
nodes: 837
nodes: 838
nodes: 840
nodes: 841
nodes: 842
nodes: 843
nodes: 844
nodes: 845
nodes: 846
nodes: 848
nodes: 849
nodes: 851
nodes: 852
nodes: 853
nodes: 854
nodes: 856
nodes: 858
nodes: 860
nodes: 861
nodes: 862
nodes: 863
nodes: 864
nodes: 866
nodes: 867
nodes: 869
nodes: 870
nodes: 872
nodes: 874
nodes: 875
nodes: 877
nodes: 879
nodes: 881
nodes: 883
nodes: 884
nodes: 886
nodes: 887
nodes: 889
nodes: 890
nodes: 891
nodes: 893
nodes: 895
nodes: 896
nodes: 898
nodes: 899
nodes: 900
nodes: 901
nodes: 902
nodes: 904
nodes: 905
nodes: 906
nodes: 908
nodes: 909
nodes: 910
nodes: 911
nodes: 912
nodes: 913
nodes: 915
nodes: 917
nodes: 918
nodes: 920
nodes: 921
nodes: 923
nodes: 924
nodes: 926
nodes: 928
nodes: 930
nodes: 932
nodes: 933
nodes: 934
nodes: 936
nodes: 937
nodes: 939
nodes: 940
nodes: 941
nodes: 942
nodes: 944
nodes: 946
nodes: 948
nodes: 949
nodes: 951
nodes: 952
nodes: 953
nodes: 954
nodes: 955
nodes: 956
nodes: 957
nodes: 959
nodes: 960
nodes: 962
nodes: 964
nodes: 965
nodes: 966
nodes: 967
nodes: 968
nodes: 969
nodes: 971
nodes: 973
nodes: 975
nodes: 976
nodes: 978
nodes: 980
