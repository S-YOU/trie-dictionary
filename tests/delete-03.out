word number: 0
NO
word number: 1
NO
word number: 2
NO
word number: 3
NO
word number: 4
NO
word number: 5
NO
word number: 6
NO
word number: 7
NO
word number: 8
NO
word number: 9
NO
word number: 10
NO
word number: 11
NO
word number: 12
NO
word number: 13
NO
word number: 14
NO
word number: 15
NO
word number: 16
NO
word number: 17
NO
word number: 18
NO
word number: 19
NO
word number: 20
NO
word number: 21
NO
word number: 22
NO
word number: 23
NO
word number: 24
NO
word number: 25
NO
word number: 26
NO
word number: 27
NO
word number: 28
NO
word number: 29
NO
word number: 30
NO
ignored
NO
word number: 31
NO
word number: 32
NO
word number: 33
NO
word number: 34
NO
word number: 35
YES
word number: 36
NO
word number: 37
NO
word number: 38
NO
word number: 39
NO
word number: 40
NO
word number: 41
NO
word number: 42
NO
word number: 43
YES
word number: 44
NO
word number: 45
NO
word number: 46
NO
word number: 47
NO
word number: 48
NO
word number: 49
NO
word number: 50
NO
word number: 51
NO
word number: 52
NO
word number: 53
NO
word number: 54
NO
word number: 55
NO
word number: 56
NO
word number: 57
NO
word number: 58
NO
word number: 59
NO
word number: 60
NO
word number: 61
NO
word number: 62
NO
word number: 63
YES
word number: 64
NO
word number: 65
NO
word number: 66
NO
word number: 67
NO
word number: 68
NO
word number: 69
YES
word number: 70
NO
word number: 71
NO
word number: 72
NO
word number: 73
NO
word number: 74
NO
word number: 75
NO
word number: 76
NO
word number: 77
NO
word number: 78
NO
word number: 79
NO
word number: 80
NO
word number: 81
NO
word number: 82
YES
word number: 83
NO
deleted: 14
word number: 84
NO
word number: 85
NO
word number: 86
NO
word number: 87
NO
word number: 88
NO
word number: 89
NO
word number: 90
NO
word number: 91
NO
word number: 92
NO
word number: 93
NO
word number: 94
NO
word number: 95
NO
word number: 96
NO
word number: 97
NO
word number: 98
NO
word number: 99
NO
word number: 100
NO
word number: 101
NO
word number: 102
NO
word number: 103
NO
word number: 104
NO
word number: 105
YES
word number: 106
NO
word number: 107
NO
word number: 108
YES
word number: 109
NO
word number: 110
NO
word number: 111
NO
word number: 112
NO
word number: 113
NO
word number: 114
NO
word number: 115
NO
word number: 116
NO
word number: 117
NO
word number: 118
NO
word number: 119
NO
word number: 120
NO
word number: 121
NO
word number: 122
NO
word number: 123
NO
word number: 124
NO
deleted: 42
word number: 125
deleted: 8
NO
word number: 126
NO
word number: 127
YES
word number: 128
NO
word number: 129
NO
word number: 130
NO
word number: 131
NO
word number: 132
YES
word number: 133
YES
word number: 134
deleted: 13
NO
word number: 135
NO
word number: 136
YES
word number: 137
NO
word number: 138
NO
word number: 139
NO
word number: 140
NO
word number: 141
YES
word number: 142
NO
word number: 143
YES
word number: 144
NO
deleted: 0
deleted: 4
deleted: 98
ignored
deleted: 15
word number: 145
NO
word number: 146
YES
word number: 147
NO
word number: 148
YES
word number: 149
NO
ignored
word number: 150
NO
word number: 151
NO
word number: 152
NO
word number: 153
NO
word number: 154
NO
word number: 155
NO
word number: 156
NO
word number: 157
NO
word number: 158
NO
word number: 159
NO
word number: 160
NO
word number: 161
NO
word number: 162
NO
word number: 163
NO
word number: 164
NO
word number: 165
NO
word number: 166
NO
word number: 167
NO
word number: 168
NO
word number: 169
NO
word number: 170
NO
word number: 171
NO
word number: 172
NO
word number: 173
NO
word number: 174
NO
word number: 175
NO
word number: 176
YES
word number: 177
YES
word number: 178
NO
word number: 179
YES
word number: 180
NO
word number: 181
NO
word number: 182
YES
word number: 183
NO
word number: 184
NO
word number: 185
YES
word number: 186
YES
word number: 187
YES
word number: 188
NO
word number: 189
NO
word number: 190
YES
word number: 191
YES
deleted: 19
word number: 192
NO
word number: 193
YES
word number: 194
NO
word number: 195
NO
word number: 196
NO
word number: 197
YES
word number: 198
YES
word number: 199
YES
word number: 200
NO
word number: 201
NO
word number: 202
NO
word number: 203
YES
word number: 204
NO
word number: 205
NO
word number: 206
NO
word number: 207
NO
word number: 208
YES
word number: 209
NO
word number: 210
NO
word number: 211
NO
word number: 212
NO
word number: 213
NO
word number: 214
NO
word number: 215
NO
word number: 216
NO
word number: 217
NO
word number: 218
YES
word number: 219
NO
word number: 220
NO
word number: 221
NO
word number: 222
NO
word number: 223
NO
word number: 224
NO
word number: 225
NO
word number: 226
YES
word number: 227
YES
word number: 228
YES
word number: 229
YES
word number: 230
NO
ignored
NO
word number: 231
YES
word number: 232
NO
word number: 233
YES
word number: 234
NO
word number: 235
NO
word number: 236
NO
word number: 237
NO
word number: 238
NO
word number: 239
NO
word number: 240
NO
word number: 241
NO
word number: 242
NO
word number: 243
NO
word number: 244
YES
word number: 245
deleted: 142
YES
word number: 246
NO
word number: 247
NO
word number: 248
NO
word number: 249
NO
word number: 250
NO
word number: 251
NO
word number: 252
YES
word number: 253
NO
word number: 254
YES
word number: 255
NO
word number: 256
NO
word number: 257
NO
word number: 258
NO
word number: 259
NO
word number: 260
NO
word number: 261
NO
word number: 262
YES
word number: 263
NO
word number: 264
NO
word number: 265
NO
word number: 266
deleted: 82
YES
word number: 267
NO
word number: 268
YES
word number: 269
NO
word number: 270
YES
word number: 271
YES
word number: 272
NO
word number: 273
NO
word number: 274
NO
word number: 275
NO
word number: 276
NO
word number: 277
NO
word number: 278
NO
word number: 279
NO
word number: 280
NO
word number: 281
NO
word number: 282
NO
word number: 283
NO
word number: 284
YES
word number: 285
YES
word number: 286
NO
word number: 287
YES
word number: 288
YES
word number: 289
YES
word number: 290
NO
word number: 291
YES
word number: 292
YES
word number: 293
YES
word number: 294
NO
word number: 295
NO
word number: 296
NO
word number: 297
NO
word number: 298
NO
word number: 299
NO
word number: 300
NO
word number: 301
NO
word number: 302
NO
word number: 303
YES
word number: 304
NO
word number: 305
YES
word number: 306
NO
word number: 307
deleted: 50
YES
word number: 308
ignored
ignored
deleted: 20
ignored
deleted: 18
ignored
deleted: 16
deleted: 17
NO
word number: 309
NO
word number: 310
NO
word number: 311
NO
word number: 312
YES
word number: 313
NO
word number: 314
YES
word number: 315
YES
word number: 316
YES
word number: 317
NO
word number: 318
YES
word number: 319
YES
word number: 320
NO
word number: 321
YES
word number: 322
YES
word number: 323
YES
word number: 324
NO
word number: 325
NO
word number: 326
YES
word number: 327
NO
word number: 328
YES
word number: 329
YES
word number: 330
NO
word number: 331
NO
word number: 332
YES
word number: 333
YES
word number: 334
NO
word number: 335
NO
word number: 336
YES
word number: 337
NO
word number: 338
YES
word number: 339
NO
word number: 340
NO
word number: 341
NO
word number: 342
NO
word number: 343
YES
word number: 344
NO
word number: 345
YES
word number: 346
YES
word number: 347
NO
word number: 348
YES
word number: 349
YES
word number: 350
YES
word number: 351
YES
word number: 352
NO
word number: 353
YES
word number: 354
NO
word number: 355
YES
word number: 356
NO
word number: 357
YES
word number: 358
YES
word number: 359
YES
word number: 360
YES
word number: 361
NO
word number: 362
YES
word number: 363
YES
word number: 364
NO
word number: 365
NO
deleted: 200
word number: 366
YES
word number: 367
YES
word number: 368
YES
word number: 369
YES
word number: 370
NO
word number: 371
YES
word number: 372
YES
word number: 373
YES
word number: 374
ignored
deleted: 1
ignored
deleted: 2
deleted: 3
deleted: 5
ignored
YES
word number: 375
YES
ignored
deleted: 21
deleted: 34
deleted: 55
deleted: 89
deleted: 144
deleted: 233
ignored
ignored
word number: 376
YES
word number: 377
NO
word number: 378
YES
word number: 379
YES
word number: 380
NO
word number: 381
YES
word number: 382
YES
word number: 383
YES
word number: 384
NO
word number: 385
YES
word number: 386
YES
word number: 387
NO
word number: 388
YES
word number: 389
NO
word number: 390
NO
word number: 391
YES
word number: 392
NO
word number: 393
NO
word number: 394
YES
word number: 395
NO
word number: 396
YES
word number: 397
NO
word number: 398
NO
word number: 399
YES
word number: 400
YES
word number: 401
NO
word number: 402
YES
word number: 403
YES
word number: 404
YES
word number: 405
NO
word number: 406
YES
word number: 407
NO
word number: 408
YES
word number: 409
NO
word number: 410
YES
word number: 411
NO
word number: 412
NO
word number: 413
YES
word number: 414
YES
word number: 415
NO
word number: 416
YES
word number: 417
YES
word number: 418
YES
word number: 419
NO
word number: 420
NO
word number: 421
YES
word number: 422
YES
word number: 423
YES
word number: 424
NO
word number: 425
NO
word number: 426
NO
word number: 427
YES
word number: 428
ignored
NO
word number: 429
NO
word number: 430
YES
word number: 431
NO
word number: 432
YES
word number: 433
YES
word number: 434
NO
word number: 435
YES
word number: 436
NO
word number: 437
YES
word number: 438
NO
word number: 439
YES
word number: 440
YES
word number: 441
NO
word number: 442
NO
word number: 443
NO
word number: 444
NO
word number: 445
YES
word number: 446
YES
word number: 447
NO
word number: 448
YES
word number: 449
YES
word number: 450
YES
word number: 451
YES
word number: 452
NO
word number: 453
NO
word number: 454
YES
word number: 455
NO
word number: 456
YES
word number: 457
NO
word number: 458
YES
word number: 459
NO
word number: 460
NO
word number: 461
NO
word number: 462
YES
word number: 463
NO
word number: 464
YES
word number: 465
NO
word number: 466
NO
word number: 467
NO
word number: 468
YES
word number: 469
NO
word number: 470
YES
ignored
YES
word number: 471
YES
word number: 472
YES
word number: 473
YES
word number: 474
YES
word number: 475
YES
word number: 476
NO
word number: 477
YES
word number: 478
YES
word number: 479
NO
word number: 480
NO
word number: 481
NO
word number: 482
NO
word number: 483
YES
word number: 484
NO
word number: 485
NO
word number: 486
YES
word number: 487
YES
word number: 488
YES
word number: 489
YES
word number: 490
NO
word number: 491
NO
word number: 492
NO
word number: 493
YES
word number: 494
NO
word number: 495
NO
word number: 496
YES
word number: 497
YES
word number: 498
NO
word number: 499
YES
word number: 500
NO
word number: 501
NO
word number: 502
YES
word number: 503
NO
word number: 504
YES
word number: 505
YES
word number: 506
YES
word number: 507
YES
word number: 508
YES
word number: 509
NO
word number: 510
YES
ignored
ignored
deleted: 9
ignored
deleted: 25
deleted: 36
deleted: 49
deleted: 64
deleted: 81
deleted: 100
deleted: 121
ignored
deleted: 169
deleted: 196
deleted: 225
deleted: 256
deleted: 289
deleted: 324
deleted: 361
deleted: 400
deleted: 441
deleted: 484
ignored
ignored
ignored
word number: 511
YES
word number: 512
NO
word number: 513
YES
word number: 514
NO
word number: 515
YES
word number: 516
YES
word number: 517
YES
word number: 518
YES
word number: 519
NO
word number: 520
NO
word number: 521
YES
word number: 522
YES
word number: 523
NO
word number: 524
YES
word number: 525
YES
word number: 526
YES
word number: 527
YES
word number: 528
YES
word number: 529
YES
word number: 530
YES
word number: 531
NO
word number: 532
NO
word number: 533
NO
word number: 534
YES
word number: 535
YES
word number: 536
YES
word number: 537
YES
word number: 538
YES
word number: 539
NO
word number: 540
YES
word number: 541
YES
word number: 542
YES
word number: 543
YES
word number: 544
YES
word number: 545
YES
word number: 546
YES
word number: 547
YES
word number: 548
YES
word number: 549
NO
word number: 550
YES
word number: 551
NO
word number: 552
YES
word number: 553
NO
word number: 554
YES
word number: 555
NO
word number: 556
NO
word number: 557
YES
word number: 558
YES
word number: 559
NO
word number: 560
YES
word number: 561
YES
word number: 562
YES
word number: 563
YES
word number: 564
YES
word number: 565
NO
word number: 566
YES
word number: 567
NO
word number: 568
YES
word number: 569
NO
word number: 570
YES
word number: 571
YES
word number: 572
YES
word number: 573
YES
word number: 574
NO
word number: 575
YES
word number: 576
NO
word number: 577
YES
word number: 578
YES
word number: 579
YES
word number: 580
YES
word number: 581
YES
word number: 582
YES
word number: 583
YES
word number: 584
YES
word number: 585
NO
word number: 586
YES
word number: 587
YES
word number: 588
YES
word number: 589
YES
word number: 590
YES
word number: 591
YES
word number: 592
YES
word number: 593
NO
word number: 594
YES
word number: 595
YES
word number: 596
NO
word number: 597
YES
word number: 598
YES
word number: 599
YES
word number: 600
YES
word number: 601
YES
word number: 602
YES
word number: 603
YES
word number: 604
YES
word number: 605
YES
word number: 606
YES
word number: 607
YES
word number: 608
NO
word number: 609
YES
word number: 610
YES
word number: 611
YES
word number: 612
YES
word number: 613
NO
word number: 614
YES
word number: 615
YES
word number: 616
YES
word number: 617
NO
word number: 618
NO
word number: 619
YES
word number: 620
NO
word number: 621
YES
word number: 622
YES
word number: 623
NO
word number: 624
NO
word number: 625
YES
word number: 626
YES
word number: 627
YES
word number: 628
YES
word number: 629
YES
word number: 630
YES
word number: 631
YES
word number: 632
YES
word number: 633
NO
word number: 634
NO
word number: 635
NO
word number: 636
YES
word number: 637
YES
word number: 638
YES
word number: 639
YES
word number: 640
YES
word number: 641
YES
word number: 642
YES
word number: 643
YES
word number: 644
NO
word number: 645
YES
word number: 646
YES
word number: 647
YES
word number: 648
YES
word number: 649
YES
word number: 650
YES
word number: 651
NO
word number: 652
YES
word number: 653
YES
word number: 654
YES
word number: 655
YES
word number: 656
YES
word number: 657
YES
word number: 658
YES
word number: 659
YES
word number: 660
YES
word number: 661
YES
word number: 662
NO
word number: 663
YES
word number: 664
YES
word number: 665
YES
word number: 666
YES
word number: 667
YES
word number: 668
YES
word number: 669
YES
word number: 670
YES
word number: 671
YES
word number: 672
YES
word number: 673
NO
word number: 674
YES
word number: 675
YES
word number: 676
YES
word number: 677
YES
word number: 678
YES
word number: 679
YES
word number: 680
YES
word number: 681
NO
word number: 682
YES
word number: 683
YES
word number: 684
YES
word number: 685
YES
word number: 686
NO
word number: 687
YES
word number: 688
YES
word number: 689
NO
word number: 690
YES
word number: 691
YES
word number: 692
YES
word number: 693
NO
word number: 694
NO
word number: 695
YES
word number: 696
NO
word number: 697
YES
word number: 698
NO
word number: 699
YES
word number: 700
YES
word number: 701
YES
word number: 702
YES
word number: 703
YES
word number: 704
YES
word number: 705
YES
word number: 706
YES
word number: 707
YES
word number: 708
YES
word number: 709
YES
word number: 710
YES
word number: 711
NO
word number: 712
YES
word number: 713
YES
word number: 714
NO
word number: 715
YES
word number: 716
YES
